ui/components/app/signature-request-siwe/index.scss

```diff
@@ -0,0 +1,140 @@
+@import 'signature-request-header/index';
+@import 'signature-request-message/index';
+
+.signature-request {
+  display: flex;
+  flex: 1 1 auto;
+  flex-direction: column;
+  min-width: 0;
+
+  @media screen and (min-width: $break-large) {
+    flex: initial;
+  }
+}
+
+.signature-request-header {
+  flex: 1;
+
+  .network-display {
+    justify-content: flex-end;
+    margin-left: auto;
+  }
+}
+
+.signature-request-content {
```

Comment:
Not sure these styles are used at all? Did a search for the class name in the component folder and it only came back with this instance

![Screen Shot 2022-04-26 at 5 59 53 PM](https://user-images.githubusercontent.com/8112138/165417818-f2814e21-700f-4210-9bb3-6c8c62e50e4f.png)

Reply:
Removed